[2023-05-11T13:24:13.359+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=test_mongo/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=upload_to_mongodb_task_2 permission to 509
[2023-05-11T13:24:13.464+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=test_mongo/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=upload_to_mongodb_task_2 permission to 509
[2023-05-11T13:24:13.591+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-05-11T13:24:13.634+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_mongo.upload_to_mongodb_task_2 scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-11T13:24:13.712+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_mongo.upload_to_mongodb_task_2 scheduled__2023-05-10T00:00:00+00:00 [queued]>
[2023-05-11T13:24:13.727+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2023-05-11T13:24:13.898+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): upload_to_mongodb_task_2> on 2023-05-10 00:00:00+00:00
[2023-05-11T13:24:13.930+0000] {standard_task_runner.py:57} INFO - Started process 887 to run task
[2023-05-11T13:24:13.940+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'test_mongo', 'upload_to_mongodb_task_2', 'scheduled__2023-05-10T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/try_connection_to_mongo.py', '--cfg-path', '/tmp/tmpzm8fhyhh']
[2023-05-11T13:24:13.948+0000] {standard_task_runner.py:85} INFO - Job 41: Subtask upload_to_mongodb_task_2
[2023-05-11T13:24:14.000+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-05-11T13:24:14.151+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=test_mongo/run_id=scheduled__2023-05-10T00:00:00+00:00/task_id=upload_to_mongodb_task_2 permission to 509
[2023-05-11T13:24:14.159+0000] {task_command.py:410} INFO - Running <TaskInstance: test_mongo.upload_to_mongodb_task_2 scheduled__2023-05-10T00:00:00+00:00 [running]> on host d8f273d8514c
[2023-05-11T13:24:14.211+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-05-11T13:24:14.532+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_mongo' AIRFLOW_CTX_TASK_ID='upload_to_mongodb_task_2' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T00:00:00+00:00'
[2023-05-11T13:24:14.560+0000] {base.py:73} INFO - Using connection ID 'mongo_default' for task execution.
[2023-05-11T13:24:14.980+0000] {logging_mixin.py:149} INFO - Connected to MongoDB - ['tiktok_collection']
[2023-05-11T13:24:15.088+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-11T13:24:15.244+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=test_mongo, task_id=upload_to_mongodb_task_2, execution_date=20230510T000000, start_date=20230511T132413, end_date=20230511T132415
[2023-05-11T13:24:15.405+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-11T13:24:15.485+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
